packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -33,7 +37,12 @@ const POPOVER_PROPS = {
 };
 
 function CopyMenuItem( { blocks, onCopy } ) {
-	const ref = useCopyToClipboard( () => serialize( blocks ), onCopy );
+	const ref = useCopyToClipboard( () => {
+		blocks = blocks.map( ( block ) =>
+			__experimentalCloneSanitizedBlock( block )
```

Comment:
I really like this suggestion, especially because it provides a better answer than I was able to give @gziolo re: serialization. And +1 for concerns on 'sanitize'; I struggled to come up with a better name and would be much happier to avoid it!